### DISCUSSION TITLE
Translate

### DISCUSSION THREAD

**Discussion by FewerElk** [ScreamingSandals/BedWars#471] [Q&A] [answered]
HI, 
I want to contribute to this project with translating en_us to fr.
I ´ve already forked this but I don't know we're I ´ve to work.
May you tell me this please ?

**Comment by zlataovce** [comment, marked as answer]
https://weblate.screamingsandals.org

**Comment by FewerElk** [comment]
And for have the translation in the server ?

**Comment by Misat11** [comment]
Open config.yml and set `locale` to the correct value. If you have made any changes to 0.2.x translations, it will appear in next version. If you want to use the current version of the language, simply download https://github.com/ScreamingSandals/BedWars/blob/ver/0.2.x/plugin/src/main/resources/languages/language_fr.yml and put it in languages folder. However, this file is currently up to date with the file bundled with BedWars 0.2.28.
